### PR TITLE
Create Post viewing post page

### PR DESCRIPTION
**Pull request by Can-Tina**

### CODE REVIEW

File: src/client/components/Post.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import "../styling/post.css"
+
+const Post = () => {
+    return (
+        <main className="postPage">
```
Comment (nathanSamosa) [inline src/client/components/Post.jsx:5]: very clean and readable layout, descriptive classNames :)

File: src/client/styling/post.css [hunk 2]
```diff
@@ -0,0 +1,69 @@
+.postPage {
+    margin: 50px 50px 50px;
+}
+
+.post {
+    box-shadow: -2px -2px 5px rgba(var(--clr-pokeblue), 0.5), 
+                2px 2px 5px rgba(var(--clr-pokeorange), 0.5);
+    padding: 20px 20px 20px
+}
```
Comment (nathanSamosa) [inline src/client/styling/post.css:9]: nice box shadow... with variables!

File: src/client/components/Post.jsx (hunk 1 above)
Comment (Can-Tina) [reply src/client/components/Post.jsx:5]: If we have time, we can change the camelCase to the dashes and stuff

File: src/client/styling/post.css (hunk 2 above)
Comment (Can-Tina) [reply src/client/styling/post.css:9]: Nice variables :)

File: src/client/styling/post.css [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+.commentDateTime {
+    margin-bottom: 7px;
+}
+
+.commentContent {
+    font-size: x-small;
+}
+
+.commentUsername {
+    margin-bottom: 5px;
```
Comment (TheunisK) [inline src/client/styling/post.css:68]: I can only dream of having your css eye Lenni. Great stuff!

File: src/client/components/Post.jsx (hunk 1 above)
Comment (TheunisK) [reply src/client/components/Post.jsx:5]: Not a problem with me I like CC.

File: src/client/components/Post.jsx [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+                <p className="postUsername">Username</p>
+                <p className="postDateTime">28/02/2022 1:18pm</p>
+                <p className="postContent">
+                    Okay so some YouTubers have pointed the star icon in the Japanese titles for the games and how said icons always hint at the new game gimmick mechanic.
+
+                    And the only thing that comes to my mind involving stars are constellations which in turn makes me think of Saint Seiya/Knights of the Zodiac, which is a series about warriors donning armor based on constellations.
+
+                    So my theory is that certain Pokemon will be able to evolve or form change similar to Zamazenta and Zacian, wearing 'armor' made of some kind. OR it's a total form change, acquiring a new typing.
+
+                    But that's my theory. What are you guys' theories.
```
Comment (vherus) [inline src/client/components/Post.jsx:16]: How are you going to make sure this `Post` component can be re-used for other posts?

**Review by vherus — APPROVED** [review]
Nice work, one point to think about (on the off-chance you haven't already)

File: src/client/components/Post.jsx (hunk 4 above)
Comment (Can-Tina) [reply src/client/components/Post.jsx:16]: There will only be 1 post per page. So it'll be a route like /posts/1 and the single post component will be used for the single post

**Pull request merged**